emerald-cli/src/main.rs

```diff
@@ -10,12 +10,12 @@ extern crate log;
 
 extern crate docopt;
 extern crate env_logger;
-extern crate emerald;
+extern crate emerald_core;
```

Comment:
I do argue that `extern crate emerald_core as emerald` or something like that would make the CLI code much clearer. But that's up to you.